docs/transpile/index.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: Introduction to the transpiler
+description: Introduction to the transpiler
+
+---
+
+
+# Introduction to the transpiler
+
+Transpilation is the process of rewriting a given input circuit to match the topology of a specific quantum device and optimize the circuit instructions for execution on present-day noisy quantum systems.  
+
+A central component of Qiskit, the transpiler is designed for modularity and extensibility. The goal is to be able to easily write new circuit transformations (known as transpiler **passes**), and combine them with other existing passes.  Which passes are chained together and in which order has a major effect on the final outcome. This pipeline is determined by a **pass manager**, which schedules the passes and also allows passes to communicate with each other by providing a shared space. In this way, the transpiler opens up the door for research into aggressive optimization of quantum circuits.
+
+This documentation covers the tooling and workflows available to Qiskit users.  If you're using primitives and are only interested in st using the default transpilation options, look to the section [Configure error suppression for Qiskit Runtime.](../run/configure-error-suppression)
+
+
+## Transformation passes
+For more advanced users, it is important to understand the various representations of quantum circuits that are used for the transpilation step along with the transformations (i.e., *passes*) that affect the efficiency of the final circuit that is sent to the backend.  This involves rewriting quantum circuits to match hardware constraints and suppressing the effects of noise.  The flow of logic for this tool chain is quite customizable and need not be linear.  One can prepare iterative loops, conditional branches, and other complex behaviors.  A good starting place when developing your own set of passes is by looking to the default sequence of transformations:
```

Comment:
I think this section should be reversed 🤔  Currently it starts off by talking about things advanced users need to know and ends with talking about the simpler defaults. I think a more natural flow would be to start with the default transformations, and then go on to talk about how advanced users can build on that knowledge to then create their own passes etc.

This is the first page where beginners are introduced to these default sequences as well as the advanced users, so i think it's worth highlighting for the beginners as well

Reply:
That makes sense.  I've switched things around now, though there's a broken link to the api and need help fixing the link.

I'm also wondering if we have a practice (or would be a good idea to implement a practice) of linking to the api reference whenever we mention a particular object or function.